students/pietrewicz_bartosz/lesson_04_unit_testing/test_collatz.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from the_collatz_sequence import collatz
+
+
+class TestCollatzFunction(unittest.TestCase):
+    def test_invalid_type(self):
+        self.assertRaises(TypeError, collatz, 'aoeu')
+
+    def test_value_8(self):
+        self.assertEquals(collatz(8), 4)
```

Comment:
assertEquals is deprecated - use assertEqual instead :) 
Have a look at:
https://docs.python.org/3.6/library/unittest.html#deprecated-aliases